docs/community/configuration.rst

```diff
@@ -0,0 +1,102 @@
+Configuration for compiling GluonNLP website
+-----------------------------------------------------------------
+
+ | The GluonNLP docs website is at https://gluon-nlp.mxnet.io (release branch), or https://gluon-nlp.mxnet.io/master/index.html (master branch). Its source code is at https://github.com/dmlc/gluon-nlp .
+
+Currently the GluonNLP website is constructed from the source code via CI automatically. Nevertheless, if a maintainer wants to make some modification and preview its changes, it will become difficult. I am writing this documentation to note down how to compile the entire website locally for future references.
```

Comment:
<del>Nevertheless, if a maintainer wants to make some modification and preview its changes, it will become difficult. I am writing this documentation to note down how to compile the entire website locally for future references.</del>
Here I will share:
- the structure of files used for the website, and
- how to make changes to the website and preview the website